app/views/rooms/show.html.erb

```diff
@@ -147,6 +147,45 @@
                 <span class="pull-right">Per Night</span>
             </div>
             <div class="card-body">
+            <!--<p>Coordinates: <%= @room.latitude %> <%= @room.longitude %></p>-->
+             <%= form_for([@room, @reservation]) do |f| %>
+
+                <div class="row">
+                    <div class="col-6">
+                        <label>Check In</label>
+                        <%= f.text_field :start_date, placeholder: "Start Date", class: "form-control" %>
+                    </div>
+                    <div class="col-6">
+                        <label>Check Out</label>
+                        <%= f.text_field :end_date, placeholder: "End Date", class: "form-control", disabled: true %>
+                    </div>
+                    </div class="col-12">
+                    <%= f.submit "Book Now", id: "button_book", class: "btn btn-primary mt-3 w-100", disabled: true %>
+                </div>
+                    <%= f.hidden_field :room_id, value: @room.id %>
+                    <%= f.hidden_field :price, value: @room.price %>
```

Comment:
Remove the room price because you already have the room_id. You can retrieve the room price via room_id